lib/google_geocoding_api.ex

```diff
@@ -67,13 +67,11 @@ defmodule GoogleGeocodingApi do
     if result, do: List.first(result["results"])["types"]
   end
 
-  defp make_request(address, opts \\ []) do
-    params = %{
-      address: address,
-      region: Keyword.get(opts, :region, "")
-    }
-
-    if key(), do: params = Map.put(params, :key, key())
+  defp make_request(address, opts) do
```

Comment:
Ahh good call, because this is a private function we'll always know to pass the opts in. Good catch :)